install/apm/php-fpm/install.yml

```diff
@@ -0,0 +1,26 @@
+id: php-fpm-agent
```

Comment:
this file looks good, but I'd recommend nesting this under the `php` directory rather than `php-fpm`. And then move the existing php install plan to a new name. Something like this would work:

```
install/apm/php/php-targeted-install/install.yml
install/apm/php/php-setup-nerdlet/install.yml
```